pkgs/development/python-modules/unblob-native/default.nix

```diff
@@ -0,0 +1,45 @@
+{
+  lib,
+  stdenv,
+  buildPythonPackage,
+  fetchFromGitHub,
+  gitUpdater,
+  rustPlatform,
+  libiconv,
+}:
+
+buildPythonPackage rec {
+  pname = "unblob-native";
+  version = "0.1.5";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "onekey-sec";
+    repo = "unblob-native";
+    tag = "v${version}";
+    hash = "sha256-jpaBxKuQNfU0I3kCs67mM5dzGURSSHvqymhk43P7xXk";
+  };
+
+  cargoDeps = rustPlatform.importCargoLock {
+    lockFile = ./Cargo.lock;
+  };
```

Comment:
Is this lockfile still necessary? We should avoid vendoring lockfiles if possible with mechanisms like `fetchCargoVendor`

Reply:
changed to use `fetchCargoVendor`